Framework7/features/step_definitions/task_steps/task_steps.rb

```diff
@@ -0,0 +1,29 @@
+require_relative '../../../src/helpers/rest_client/api_rest_client'
+require_relative '../../../src/data/task'
+
+Given(/^I have set a connection to pivotal tracker API service$/) do
+  @client=ApiRestClient.new
+end
+
+Given(/^I have ([1-9]{7}) from  Pivotal Tracker project the project$/) do |project_id|
+  @project_id=project_id
+end
+
+Given(/^I have ([0-9]{9}) from  Pivotal Tracker project the story$/) do |story_id|
+  @story_id=story_id
+end
+Given(/^I have (.*?) the task name$/) do|description_task|
+   p @description_task=description_task
```

Comment:
try to clean the print lines in the code that you submit only @description_task=description_task should be ok
Print only should be used in your development